src/site/content/en/fast/fetch-priority/index.md

```diff
@@ -1,14 +1,15 @@
 ---
 layout: post
-title: Optimizing resource loading with Priority Hints
+title: Optimizing resource loading with the Fetch Priority API
 authors:
   - leenasohoni
   - addyosmani
   - patmeenan
-description: Priority Hints indicate the relative priority of resources to the browser. They can enable optimal loading and improve Core Web Vitals.
-subhead: Priority Hints indicate the relative priority of resources to the browser. They can enable optimal loading and improve Core Web Vitals.
+  - tunetheweb
+description: The Fetch Priority API indicates the relative priority of resources to the browser. They can enable optimal loading and improve Core Web Vitals.
+subhead: The Fetch Priority API indicates the relative priority of resources to the browser. They can enable optimal loading and improve Core Web Vitals.
```

Comment:
```suggestion
description: The Fetch Priority API indicates the relative priority of resources to the browser. It can enable optimal loading and improve Core Web Vitals.
subhead: The Fetch Priority API indicates the relative priority of resources to the browser. It can enable optimal loading and improve Core Web Vitals.
```